reactor-core/src/main/java/reactor/core/publisher/ImmutableSignal.java

```diff
@@ -69,4 +70,72 @@ public SignalType getType() {
 		return type;
 	}
 
+	//the base class defines equals and hashcode as final in order to allow
+
```

Comment:
equals is not final and this is not a base class anymore, so remove broken comment ;)